portality/templates/application_form/_value.html

```diff
@@ -0,0 +1,15 @@
+{% macro value(x, field_name, formulaic_context) %}
+    {% if x == None or x == "" or x == [] %}
+        <em>[Nothing]</em>
+    {% elif x is string and x.startswith("http") %}
+        <a href="{{ x }}" target="_blank" rel="noopener" style="word-break: break-all">{{ x }}</a>
```

Comment:
should embedded style actually be in the SASS?

Reply:
I did that one. This is because we don’t want to `break-all` link styles necessarily. Inlining isn’t always a bad thing when you need to apply a style to a specific element (plus, in certain cases, it is better for performance).